src/components/earth/Warning.tsx

```diff
@@ -0,0 +1,132 @@
+// src/components/common/WarningTooltip.tsx
+import { useState } from "react";
+import styled, { keyframes } from "styled-components";
+
+import WarningIcon from "../../assets/svgs/Warning.svg?react";
+import { rowFlex } from "../../styles/flexStyles";
+import theme from "../../styles/theme";
+
+const Warning = () => {
+  const [hovered, setHovered] = useState(false);
+
+  return (
+    <Wrapper>
+      <IconWrapper
+        onMouseEnter={() => setHovered(true)}
+        onMouseLeave={() => setHovered(false)}
+      >
+        <GlowLayer $visible={hovered} />
+        <WarningIconStyled />
+        <Minicard $visible={hovered}>
+          <Title>
+            South Korea <Emoji>🌧️</Emoji>
+          </Title>
+        </Minicard>
+      </IconWrapper>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
+  width: 80%;
+  height: 100%;
+  ${rowFlex({ align: "center", justify: "center" })}
+`;
+
+const IconWrapper = styled.div`
+  position: relative;
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+  svg {
+    width: 100%;
+    height: 100%;
+  }
+  &:hover > div {
+    opacity: 1;
+    transform: translateY(-12px);
+    pointer-events: auto;
+  }
+`;
+const glowFade = keyframes`
+  0% {
+    opacity: 0.6;
+    transform: scale(1);
+  }
+  50% {
+    opacity: 1;
+    transform: scale(1.2);
+  }
+  100% {
+    opacity: 0.6;
+    transform: scale(1);
+  }
+`;
+
+const GlowLayer = styled.div<{ $visible: boolean }>`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  background: radial-gradient(
+    circle,
+    rgba(255, 199, 78, 0.5) 20%,
+    transparent 70%
+  );
+  z-index: 0;
+  opacity: ${({ $visible }) => ($visible ? 1 : 0)};
+  pointer-events: none;
+  animation: ${({ $visible }) => ($visible ? glowFade : "none")} 2.5s
+    ease-in-out infinite;
+  transition: all 0.5s ease;
+`;
+
+const WarningIconStyled = styled(WarningIcon)`
+  width: 100%;
+  height: 100%;
+  z-index: 10;
+`;
+
+const Minicard = styled.div<{ $visible: boolean }>`
+  position: absolute;
+  left: 80px;
+  top: -30px;
+  width: 250px;
+  height: 120px;
+  padding: 12px 16px;
+  border-radius: 10px;
+  background: rgba(0, 8, 23, 0.75);
+  color: ${theme.colors.textPrimary};
+  white-space: nowrap;
+  font-weight: bold;
+  font-size: 16px;
+  opacity: ${({ $visible }) => ($visible ? 1 : 0)};
+  pointer-events: ${({ $visible }) => ($visible ? "auto" : "none")};
+  transform: translateY(${({ $visible }) => ($visible ? "-12px" : "0")});
+  transition: all 0.5s ease;
+  z-index: 999;
+  backdrop-filter: blur(4px);
+
+  &::after {
+    content: "";
+    position: absolute;
+    left: -6px;
+    top: 50%;
+    transform: translateY(-50%) rotate(45deg);
+    width: 30px;
+    height: 30px;
+    background: rgba(0, 8, 23, 0.75);
+    z-index: 999;
+  }
```

Comment:
```suggestion
  &::after {
    content: "";
    position: absolute;
    left: -10px;
    top: 50%;
    transform: translateY(-50%);
    border-top: 10px solid transparent;
    border-bottom: 10px solid transparent;
    border-right: 10px solid rgba(0, 8, 23, 0.75);
    border-left: none;
  }
```
말풍선을 만들 때 이렇게 직접 만드니 나중에 위치가 안맞는 문제가 생길 수도 있겠네요 😢 일단 이렇게 구현하고 나중에 라이브러리를 사용해야할 것 같아요!

Reply:
넵! 혹시 아래 제안해주신 내용은 어떤게 바뀐건가요??

Reply:
### BEFORE
![image](https://github.com/user-attachments/assets/4f5a55d0-ae9b-49fb-8aa0-2654911ec2e6)


### AFTER
![image](https://github.com/user-attachments/assets/6ea1e36d-6288-4c80-944c-b6d868052dbe)

이전에 화살표 부분이 부자연스러웠던 문제를 해결해보았습니다!